CharacterCreation/01_Races.txt

```diff
@@ -116,45 +153,73 @@ Starting Stat Points: 40
-    DC 25 Biomechanics check, or a DC 50 Mechanics check. It can be upgraded 
-    to any CQB weapon with a DC 50 Biomechanics or DC 75 Mechanics check.
-    Integrated multitool: Your fingers hide a basic toolkit, including low-
-    speed motorized screwdrivers, soldering iron, wrenches, quantum spanner, 
-    toothpick, pliers, sharp edge, lock pick, etc. If it fits in a leatherman,
-    its plausible that it would be in this toolkit.
-    Internal Reinforcing: Critical Damage threshold +5, +5 base health.
-    External Reinforcing: 5 DR passive, potential for RP abilities.
-
-Cyborgs share all weaknesses with their base race, generally humans. They 
+
+* Ocular Augmentation: +1 Perception, IR Vision 20m.
+* Enhanced Speed: +1 movement.
+* Enhanced Strength: +1 STR, metal arms, legs, or both.
+* Neural Implant: +1 Intelligence.
+* Speed Geared Fine Motor Controls: +1 DEX, no augment to movement speed.
+* Skeletal Support Structure: +1 FORT.
+* Implanted Weapon: See Below
+* Integrated multi-tool: See Below
+* Internal Reinforcing: Critical Damage threshold +5, +5 base health.
+* External Reinforcing: 5 DR passive, potential for RP abilities.
+
+    Once at level 3, and every 5 levels thereafter a Cyborg may purchase an 
+additional augment at the cost of $2000. They must see a cybernetics expert to
+have this augment installed.
+
+== Implanted Weapon ==
+
+    Pick a pistol from the basic weapons list. This pistol is implanted in a 
+limb. You may manipulate your limb to use this weapon at the cost of 1 action. 
+While the weapon is out, that limb does not operate at its normal capacities. 
+Hands would not grasp, legs would not superbly support you, etc. Subtract the 
+price of the weapon from your class starting money. The weapon can be upgraded 
+to another pistol with a DC 25 Biomechanics check, or a DC 50 Mechanics check. 
+It can be upgraded to any CQB weapon with a DC 50 Biomechanics or DC 75 
+Mechanics check.
+
+== Integrated Multi-tool ==
+
+    Your fingers hide a basic toolkit, including low-speed motorized 
+screwdrivers, soldering iron, wrenches, quantum spanner, toothpick, pliers, 
+sharp edge, lock pick, etc.
+
+===== Weaknesses =====
+
+    Cyborgs share all weaknesses with their base race, generally humans. They 
 additionally are weak against electric damage taking an extra 50% damage.(10
 damage would become 15). They are also susceptible to hacking, although they 
 would not come under control, they would merely loose the use of their 
 augmentations and any affected limbs. DC 80 Hack check.
-Once at level 3, and every 5 levels thereafter a Cyborg may purchase an 
-additional augment at the cost of $2000. They must see a cybernetics expert to
-have this augment installed.
 
-//============================================================
+==============================
 Opaleite
+==============================
+
+    Opaleites are so named by the humans who encountered them for the way the 

```

Comment:
Nonblocker: This section is unique in the document for not being attached to a section header. This is an inconsistency. Based on the paragraph in question, I conjecture it would appreciate an ===== Anatomy ===== header between 205 and 207.